src/models/team.coffee

```diff
@@ -0,0 +1,75 @@
+class Team
+
+  @robot = null
+
+  @store: ->
+    throw new Error('robot is not set up') unless @robot
+    @robot.brain.data.teams or= {}
+
+  @defaultName: ->
+    '__default__'
+
+  @all: ->
+    teams = []
+    for key, teamData of @store()
+      continue if key is @defaultName()
+      teams.push new Team(teamData.name, teamData.members)
+    teams
+
+  @default: (members = [])->
+    unless @exists @defaultName()
+      @create(@defaultName(), members)
+    @find(@defaultName())
+
+  @count: ->
+    Object.keys(@store()).length
+
+  @find: (name)->
```

Comment:
This looks like a getter too, why not simply `get`? and it should return `null` since its normal return value is not a boolean.